README.md

```diff
@@ -28,65 +17,82 @@ The key files contained in this repository -- which will also
 appear in templated copies -- are listed below, with some things
 to know about each.
 
-### README
+---
+
+## README
+
+Each repository will contain a plain-text [README file][wk-rdm],
+preferably formatted using [GitHub-flavored Markdown][gh-mdn] and
+named `README.md` (this file) or `README`.
 
-Each repository will contain a [README file][wk-rdm], preferably
-formatted using [GitHub-flavored Markdown][gh-mdn] and named
-`README.md` (this file). It must contain:
+Per the [GitHub ROB][gh-rob] and [NIST Suborder
+1801.02][nist-s-1801-02], your README should contain:
```

Comment:
The NIST Suborder (nist-s-1801-02) doesn't show up as a link when I view it. (nor does the cornell-meta link).

Reply:
Thanks! Found & fixed several typos, and added the Cornell link.